### ISSUE TITLE
Maven Dependency convergence errors

### ISSUE THREAD

**Issue by ncoelle** [fhoeben/hsac-fitnesse-fixtures#174]
Using the maven Dependency Plugin I get a lot of "Dependency convergence error" messages. Maybe you should update the POM settings and resolve the dependencies in hsac-fitnesse-fixtures.

Here an example of the error messages:

```
Dependency convergence error for org.seleniumhq.selenium:selenium-chrome-driver:3.7.0 paths to dependency are:
+-<MY COMPONENTS>
  +-nl.hsac:hsac-fitnesse-fixtures:3.8.0
    +-org.seleniumhq.selenium:selenium-java:3.7.0
      +-org.seleniumhq.selenium:selenium-chrome-driver:3.7.0
and
+-<MY COMPONENTS>
  +-nl.hsac:hsac-fitnesse-fixtures:3.8.0
    +-com.galenframework:galen-core:2.3.5
      +-org.seleniumhq.selenium:selenium-chrome-driver:3.5.3
```

I resolved them by adding the following exclusions in my pom.xml:

```
<dependency>
	<groupId>com.galenframework</groupId>
	<artifactId>galen-core</artifactId>
	<version>2.3.5</version>
	<exclusions>
		<exclusion>
			<groupId>com.codeborne</groupId>
			<artifactId>phantomjsdriver</artifactId>
		</exclusion>
		<exclusion>
			<groupId>net.bytebuddy</groupId>
			<artifactId>byte-buddy</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.freemarker</groupId>
			<artifactId>freemarker</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.seleniumhq.selenium</groupId>
			<artifactId>selenium-chrome-driver</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.seleniumhq.selenium</groupId>
			<artifactId>selenium-edge-driver</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.seleniumhq.selenium</groupId>
			<artifactId>selenium-firefox-driver</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.seleniumhq.selenium</groupId>
			<artifactId>selenium-java</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.seleniumhq.selenium</groupId>
			<artifactId>selenium-support</artifactId>
		</exclusion>
		<exclusion>
			<groupId>org.slf4j</groupId>
			<artifactId>slf4j-simple</artifactId>
		</exclusion>
	</exclusions>
</dependency>
```

Here the configuration of my dependency-plugin:

```
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-dependency-plugin</artifactId>
	<version>3.0.0</version>
	<executions>
		<execution>
			<id>analyze</id>
			<goals>
				<goal>analyze-only</goal>
			</goals>
			<configuration>
				<failOnWarning>true</failOnWarning>
				<outputXML>true</outputXML>
				<ignoreNonCompile>true</ignoreNonCompile>
			</configuration>
		</execution>
	</executions>
</plugin>
```

**Comment by fhoeben** [comment]
Thanks for the detailed analysis. These are not real problems, as Maven select's the 'nearest' dependency automatically (i.e. the version I state and not galen-framework's version), but potential ones, correct?

I don't mind excluding the dependencies to make it clear I believe we should be on the newer version. I will just have to double check what the current dependencies are  of galen-framework each time I upgrade its version.

Put the change you suggest in a pull request?

**Comment by fhoeben** [comment]
I believe this is resolved in 3.17.0
